src/services/userService.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+let users = [];
+
+const init = () => {
+  users.length = 0;
+};
+
+const getAllUsers = () => {
+  return users;
+};
+
+const getUserById = (id) => {
+  const userById = users.find(user => user.id === +id);
+
+  return userById;
+};
+
+const createUser = (name) => {
+  const id = Date.now();
```

Comment:
You can write a function to generate id, and use it and all places where you need it.